public/static/partials/subscription_modal.html

```diff
@@ -60,7 +60,7 @@ <h4>[[ c.verb ]] Subscription</h4>
                     </div>
                     <div ng-show="method.value === 'slack'">
                         <label for="slack">Slack Username or Channel</label>
-                        <input id="slack" ng-model="targets['slack']" placeholder="@user"></input>
+                        <input id="slack" ng-model="targets['slack']" placeholder="MEMBERID, @user or #channel"></input>
```

Comment:
Do we expect users to input memberID? I don't think they're going to know how to find that; we may want to keep this simpler.

Reply:
Does that mean the script will need to run through old notifications and update them to use memberIDs or anything like that?

Reply:
The script will simply add the memberID to all users. Old ones that have the slack userName will still work, but new ones that are created will have the memberID. I think at this point updating all old notifications seems overkill since it still works as expected.

Reply:
If we expect it to stop working as expected then it is something we'll need to do, at least for project level subscriptions, even though it sounds like we don't have a timeline on that. 

Anyway this all makes sense to me -- can we just change the text to say Member ID instead of MEMBERID since that's a bit more user-friendly as text?

Reply:
(We also may want to update in spruce; not sure if that's already been done)

Reply:
We had a similar discussion on the Spruce PR. The reason it is "MEMBERID" is to provide a hint to the user about the format of the memberID, which will look like this: ASDLY25TL.